src/app/demo/page.tsx

```diff
@@ -5,19 +5,32 @@ import Workshop from "../components/workshop";
 
 export default function Demo() {
   return (
-    <div className={styles.demo}>
+    <section id="workshops" className={styles.demo}>
       <h2 className={styles["demo__title"]}>
         ¡No Te Pierdas Nuestras Próximas Clases!
       </h2>
       <p className={styles["demo__subtitle"]}>
         Reserva Tu Lugar en Nuestros Cursos y Talleres Más Populares
       </p>
       <div className={styles.workshops}>
-        <Workshop />
-        <Workshop />
-        <Workshop />
-        <Workshop />
+        <Workshop
+        name="Taller de Git y GitHub"
+        startDate="5 de julio de 2024"
+        duration="3 dias"
+        schedule="10:00 a 13:00 hrs"
+        imageSrc={ScreenShot}
```

Comment:
En vez de usar ScreenShot, mejor escribe Screenshot, se lee mejor
```suggestion
        imageSrc={Screenshot}
```